src/util/commandline/app-command.ts

```diff
@@ -45,40 +45,8 @@ export class AppCommand extends Command {
 
     return result;
   }
-}
-
-export function getCurrentApp(optValue: string): ResultOrValue<DefaultApp> {
```

Comment:
Yea, turns out using `AppCommand` as the superclass of `list` totally doesn't make sense so I can't use `this.app` => Adding `getCurrentApp` back